test/integration/ServerTests/StartingAppLocallyTest.cs

```diff
@@ -68,6 +69,7 @@ public void StartingAndroidAppWithCapabilitiesOnTheServerSideTest()
             var clientCapabilities = new AppiumOptions();
             clientCapabilities.AddAdditionalAppiumOption(AndroidMobileCapabilityType.AppPackage, "io.appium.android.apis");
             clientCapabilities.AddAdditionalAppiumOption(AndroidMobileCapabilityType.AppActivity, ".view.WebView1");
+            clientCapabilities.AutomationName = AutomationName.AndroidUIAutomator2;
```

Comment:
@jlipps / @mykola-mokhnach, As discussed on the slack channel until now the test failed since we passed `AutomationName` only to server side caps. I assume appium expects the cap on both sides, please approve it's not a bug.